### PR TITLE
MPW-9: Links updated

### PR DESCRIPTION
**Pull request by devBoison**
#### What does this PR do?
...your answer here...
#### Where would you like the reviewer to focus their attention?
...your answer here...
#### Are any additional testing steps needed?
...your answer here...
#### Complete each item below:
- [ ] Did you include relevant tickets in the PR title?
- [ ] Did you pre-review your own PR before asking for external review?
- [ ] Did you make sure it works well?

### CODE REVIEW

**Review by josarp — APPROVED** [review]

File: assets/pages/bookmark.html [hunk 1]
```diff
@@ -104,46 +112,56 @@ <h2>Static Previews</h2>
...
+                        </svg>
+                    </div>
                 </div>
             </div>
-            </div>
         </div>
     </footer>
 </body>
+
 </html>
```
Comment (josarp) [inline assets/pages/bookmark.html:168]: please add an empty line

Comment (josarp) [reply assets/pages/bookmark.html:168]: all files should end with with a blank line. See that github is warning you?

File: assets/pages/insure.html [hunk 2]
```diff
@@ -106,46 +110,56 @@ <h2>Static Previews</h2>
...
+                        </svg>
+                    </div>
                 </div>
             </div>
-            </div>
         </div>
     </footer>
 </body>
+
 </html>
```
Comment (josarp) [inline assets/pages/insure.html:165]: add a blank line here

File: assets/pages/manage.html [hunk 3]
```diff
@@ -104,46 +108,56 @@ <h2>Static Previews</h2>
...
+                        </svg>
+                    </div>
                 </div>
             </div>
-            </div>
         </div>
     </footer>
 </body>
+
 </html>
```
Comment (josarp) [inline assets/pages/manage.html:163]: add a blank line here

File: assets/pages/porfolio.html [hunk 4]
```diff
@@ -95,46 +107,56 @@ <h2>Fylo</h2>
...
+                    </div>
                 </div>
             </div>
-            </div>
         </div>
     </footer>
 </body>
-</html>
+
+</html>
```
Comment (josarp) [inline assets/pages/porfolio.html:162]: add a blank line here

File: assets/pages/insure.html (hunk 2 above)
Comment (devBoison) [reply assets/pages/insure.html:165]: Noted.

**Pull request merged**